server/src/septic/septicMetaInfo.ts

```diff
@@ -148,13 +165,12 @@ export class SepticMetaInfoProvider {
         return objectsMap;
     }
 
-    private loadObjectsDocumentation(): Map<
-        string,
-        ISepticObjectDocumentation
-    > {
+    private loadObjectsDocumentation(
+        version: string
+    ): Map<string, ISepticObjectDocumentation> {
         const filePath = path.join(
             __dirname,
-            "../../../public/objectsDoc.yaml"
+            `../../../public/${version.replace(/\./g, "_")}/objectsDoc.yaml`
```

Comment:
Out of interest: Does the join() method properly handle joining paths with "../"? I.e. does join("/home/user", "../../root") turn into "/root" or "/home/user/../../root"?

Reply:
Yes, I think that pathlib handles "../" correctly